sktime/utils/validation/forecasting.py

```diff
@@ -328,13 +329,22 @@ def check_scoring(scoring):
     ----------
     TypeError
         if object is not callable from current scope.
+    NotImplementedError
+        if metric requires y_pred_benchmark to be passed
     """
     # Note symmetric=True is default arg for MeanAbsolutePercentageError
     from sktime.performance_metrics.forecasting import MeanAbsolutePercentageError
 
     if scoring is None:
         return MeanAbsolutePercentageError()
 
+    if _has_tag(scoring, "requires-y-pred-benchmark") and allow_y_pred_benchmark:
```

Comment:
The condition needs to be fixed here, right? 
```suggestion
    if _has_tag(scoring, "requires-y-pred-benchmark") and not allow_y_pred_benchmark:
```

Reply:
Yep -- good catch!